Omnichannel Triggers - Administration Page goes blank

### Description:

As soon as I add a LiveChat Trigger, the LiveChat Triggers admin page goes blank. The newly added triggers is added to the database, but nothing listed in the administration. Can not add a new trigger. I have to clean the triggers database table to get the admin panel back.

### Steps to reproduce:

1. /omnichannel/triggers
2. Add new trigger via +

### Expected behavior:

Add trigger and trigger shows up in the list.

### Actual behavior:

Trigger is saved, but admin page is blank. 
JS Console shows:
/node_modules/react-dom/cjs/react-dom.production.min.js:217 Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= 

This error has been there for a couple of versions. Reinstalled multiple times to make sure everything is ok codewise.

### Server Setup Information:

- Version of Rocket.Chat Server: 1.7
- Operating System: Debian
- Deployment Method: manual
- Number of Running Instances: 1
- DB Replicaset Oplog: replicaSet=rs01
- NodeJS Version: v12.18.4
- MongoDB Version: v4.0.10

### Client Setup Information

- Desktop App or Browser Version: Chrome
- Operating System: Mac OSX Mojave

### Additional context


### Relevant logs:
Js Console:
/node_modules/react-dom/cjs/react-dom.production.min.js:217 Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]=

--- a/client/omnichannel/triggers/TriggersTable.js
+++ b/client/omnichannel/triggers/TriggersTable.js
@@ -1,5 +1,5 @@
-import { Table, Callout, Icon } from '@rocket.chat/fuselage';
-import { useMutableCallback, Button } from '@rocket.chat/fuselage-hooks';
+import { Table, Callout, Icon, Button } from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
 import React, { useState, memo, useMemo } from 'react';
 
 import GenericTable from '../../components/GenericTable';

